test/fast/Aliases/lts/"nvm alias" should ensure LTS alias dir exists

```diff
@@ -7,10 +7,23 @@ set -ex
 
 LTS_ALIAS_PATH="$(nvm_alias_path)/lts"
 
-die () { echo "$@" ; exit 1; }
+if [ -d "${LTS_ALIAS_PATH}" ]; then
```

Comment:
Will `setup` and `teardown` be called for every test script? I'm sure I have no that directory but it just appears during the test, `setup` will do `rm -Rf v* src alias` but must not be every round and that's why I got that lts directory during the test which makes the test failed locally.